ansible-runner/context/app/project/roles/mirror_appliance/tasks/migrate.yaml

```diff
@@ -23,7 +23,7 @@
 
 - name: Verify sqlite3 cli binary is available as entrypoint command from the container
   command: >
-    podman run --name sqlite-cli --rm {{ sqlite_image }} --version
+    timeout 40 podman run --name sqlite-cli {{ sqlite_image }} --version
```

Comment:
is 40s special here at all? How long should this command take? I would expect fairly quickly